test/csuite/timestamp_abort/main.c

```diff
@@ -167,12 +169,17 @@ thread_ts_run(void *arg)
         testutil_assert(ret == 0 || ret == WT_NOTFOUND);
         if (ret == 0) {
             /*
-             * Set both the oldest and stable timestamp so that we don't need to maintain read
-             * availability at older timestamps.
+             * Periodically let the oldest timestamp lag.
              */
-            testutil_check(__wt_snprintf(tscfg, sizeof(tscfg),
-              "oldest_timestamp=%s,stable_timestamp=%s", ts_string, ts_string));
+            if (!first && __wt_random(&rnd) % 4 == 0)
+                testutil_check(__wt_snprintf(tscfg, sizeof(tscfg),
+                  "oldest_timestamp=%s,stable_timestamp=%s", prev_ts, ts_string));
```

Comment:
My preference here will be to just set the stable_timestamp alone, as there is no change in the value of the oldest_timestamp value.  I think that way it will be more evident that the test is not changing the oldest timestamp value. This is just a suggestion.

Reply:
I like that cleanup. And it gets rid of the memcpy and local variable.